alarms/dojot.auth.AuthorizationError.xml

```diff
@@ -0,0 +1,12 @@
+<alarm-metamodel>
+       <domain>AuthorizationError</domain>
+       <namespace>dojot.auth</namespace>
+       <primary-subject>
+               <primary-field type="alphanumeric">module_name</primary-field>
+               <primary-field type="alphanumeric">instance_id</primary-field>
+       </primary-subject>
+       <additional-data>
```

Comment:
Perhaps here we could include more useful information to an admin observing the event, like
- userid
- resource (what is this user attempting to access)